### ISSUE TITLE
Having problems with XML and all guide links erroring out.

### ISSUE THREAD

**Issue by BigBoyDale** [acidjesuz/EPGTalk#1]
Have been trying to get your EPG to download and populate into Kodi as well as perfect player and Best Player.  In the past they downloaded seamlessly.  However in the last month the links changed and the new links don't work for me.  I made changes in the links to your updated ones, but they continually error out with no guide.  I'm in the US and have tried everything with no avail.  Can you help me please with any ideas.  Thanks Spencer!  What is the exact  full link that should work for "guide.xml".  I have to be missing something.

**Comment by acidjesuz** [comment]
Hey Spencer,

This is the master link: https://raw.githubusercontent.com/acidjesuz/EPGTalk/master/guide.xml you should use in your players. 
Nothing really changed besides the link from EPG to EPGTalk. 

Let me know if the above link works for you.

aCiD

**Comment by BigBoyDale** [comment]
I tried the link this weekend with a couple of programs...Kodi v18. and
Best Player.  I can't even figure out how to put a link in Best Player,
even though somehow it miraculously. downloaded a few weeks ago.  If you
have a program that works and easily integrates with your guide, let me
know and I'll give it a try.  I'll keep trying.  I had noticed that others
on the IPTVzeta forum were having problems since the change..  Thanks for
your quick response.!  Spencer

**Comment by acidjesuz** [comment]
Hey there,

It might be something on your end, it works fine here and I don't see people reporting anything about guide not working.
Link's work all fine.

Try TiviMate,  you might like the player and it works fine with this guide.

aCiD

**Comment by BigBoyDale** [comment]
I'll give it a shot and update if I have success.  Thanks again Arthur!

**Comment by acidjesuz** [comment]
Hey BigBoy,

Quick question, what happens when you try to load the guide in Kodi on any other player you trying to use? 
Are you getting any errors? Or it just not loading at all? 

I'm trying to see if I can replicate the issue and see what's going on.

aCiD

**Comment by BigBoyDale** [comment]
Basically it just loads the channels on the top right ( I can see them scrolling).   And I have all the channels listed, but the guide never populates in.  I just see listed channels.  NO ERRORS...in Kodi ,  However, I don't know if your familiar with PerfectPlayer.  I'm trying to get it setup as a test with my providers channels.  It's pretty clunky but it has a way to test the EPG's before you load them and that program shows "error" .  I'm still fighting with that program just to get a channel list though.  Says it downloads one "OK", but I'm not sure about it as nothing appears.  I can send you screenshots of Kodi, but I don't think this editor supports images.  Just send me an email to Geezer1956@gmail.com  and I'll upload screenshots...Thanks again! Heck, I'd even let you log in to my account but you need version 18 or lower to make this add-on work..

**Comment by BigBoyDale** [comment]
Or you can log in to my machine with team viewer...

**Comment by acidjesuz** [comment]
Hey BigBoy,

I just got home from work, let me breath a little and I'll see what can be done about it. I'll install Kodi and PerfectPlayer and see if I can get it going.

I deleted your email from the post (you don't want to give it away your personal email here).

I'll get back to you soon.

aCiD

**Comment by BigBoyDale** [comment]
Thanks and if you're tired, no big rush.  We can do it whenever you have
some time.   If you need to login in to my computer via Team Viewer, no
problem.  I'd like to get it figured out.  Perfect player has not been that
"perfect" for me.   Or if you need a screenshot and can send it to you via
Gmail, let me know.  Spencer

**Comment by acidjesuz** [comment]
Personally I never used Perfect Player before, I used Kodi sometime ago but then I moved to TiviMate and never looked back. 
I know it cost few bucks, but in my opinion it's worth it every single penny.

What add-on are you using with Kodi for iptv?

aCiD

**Comment by BigBoyDale** [comment]
It's IPTV talk.  This was an add on that was written by a guy on my iptv
provider's site.  Unfortunately he abandoned the project in about 2018, but
it still worked fine. I can send you the add-on, but again it has to run in
v18 or earlier.  I'll look for it.

**Comment by BigBoyDale** [comment]
I'll try to  send  the zip.
[plugin.video.iptvxtra2-16.5.zip](https://github.com/acidjesuz/EPGTalk/files/14016933/plugin.video.iptvxtra2-16.5.zip)

**Comment by acidjesuz** [comment]
Good morning, 

So last night I was trying to get it going but to no avail, I tried few different versions of Kodi but still no go. 
At first it was loading the list but then it stopped, so I don't know if that's something with Kodi or what. I tried 18 thru 19 something to no avail.

I haven't used Kodi in years so I have no idea what might be wrong. 

Do you really need Kodi?

aCiD

**Comment by BigBoyDale** [comment]
No..I also use this program and it used to download the epg also.
Sometimes you do have to restart the program after changing channels, and
this occured after the provider updated their servers.  But it's well
written.  I am going to look at TiVimate, but it looks like one has to use
an emulator in order to run in Windows.
https://best-player.eu/help/epg-track-association/
It says you have to pay for it, but I downloaded it and it still works
free.

**Comment by acidjesuz** [comment]
You can use BlueStacks as emulator in Windows, it works pretty good.
Take a look at Tivimate I don't think you will be disappointed. 

Or just maybe get a small Android Box they're not expensive and that way you wouldn't have to run your PC all the time.

aCiD
